pkg/virtual/framework/forwardingregistry/store.go

```diff
@@ -87,9 +92,115 @@ func DefaultDynamicDelegatedStoreFuncs(
 
 		return delegate.Get(ctx, name, *options, subResources...)
 	}
-	s.CreaterFunc = nil           // not currently supported
-	s.GracefulDeleterFunc = nil   // not currently supported
-	s.CollectionDeleterFunc = nil // not currently supported
+	s.CreaterFunc = func(ctx context.Context, obj runtime.Object, createValidation rest.ValidateObjectFunc, options *metav1.CreateOptions) (runtime.Object, error) {
+		err := rest.BeforeCreate(createStrategy, ctx, obj)
+		if err != nil {
+			return nil, err
+		}
+		if createValidation != nil {
+			if err := createValidation(ctx, obj.DeepCopyObject()); err != nil {
+				return nil, err
+			}
+		}
+		unstructuredObj, ok := obj.(*unstructured.Unstructured)
+		if !ok {
+			return nil, fmt.Errorf("not an Unstructured: %#v", obj)
+		}
+		delegate, err := client(ctx)
+		if err != nil {
+			return nil, err
+		}
+		return delegate.Create(ctx, unstructuredObj, *options, subResources...)
+	}
+	s.GracefulDeleterFunc = func(ctx context.Context, name string, deleteValidation rest.ValidateObjectFunc, options *metav1.DeleteOptions) (runtime.Object, bool, error) {
+		obj, err := s.Get(ctx, name, &metav1.GetOptions{})
+		if err != nil {
+			return nil, false, err
+		}
+		graceful, gracefulPending, err := rest.BeforeDelete(deleteStrategy, ctx, obj, options)
+		if err != nil {
+			return nil, false, err
+		}
+		if deleteValidation != nil {
+			err = deleteValidation(ctx, obj.DeepCopyObject())
+			if err != nil {
+				return nil, false, err
+			}
+		}
+		delegate, err := client(ctx)
+		if err != nil {
+			return nil, false, err
+		}
+		err = delegate.Delete(ctx, name, *options, subResources...)
```

Comment:
don't we need some uid+rv pre-condition?

Reply:
The options `UID` and `ResourceVersion` pre-conditions are checked in the `rest.BeforeDelete` function call few lines above.

Reply:
I've added some concurrency control logic that defaults the UID and ResourceVersion pre-conditions in 0c9c4d96fe77108dfb1d5c16eb514bb7024b76f7. Let me know if this is what you have in mind.